cli/replicasets/expel.go

```diff
@@ -64,6 +65,10 @@ func Expel(ctx *context.Ctx, args []string) error {
 
 	for instanceName, instanceFound := range instancesToExpelMap {
 		if !instanceFound {
+			if err := common.CheckInstanceNameTypoWithProjectName(instanceName, ctx.Project.Name,
+				fmt.Sprintf("Instance %s isn't found in cluster: ", instanceName)); err != nil {
```

Comment:
Yes. I think we may simply return the error in wrapper. Wrapper should add useful comment, if it makes sence, and return the same unwrapped error otherwise.

Reply:
Done